common/src/main/webapp/less/style.less

```diff
@@ -1089,19 +1089,24 @@ tfoot {
 @mobile-responsive-cutoff: 670px;
 
 .target-form {
-  grid-template-columns: [label] minmax(100px, 1fr) [field] 6fr;
+  grid-template-columns: [label] minmax(100px, 10%) [field] 1fr;
   justify-items: stretch;
 }
 
 .target-search-form {
   grid-column: 1 / span 2;
   display: grid;
-  grid-template-columns: [label] minmax(100px, 1fr) [field] 6fr;
+  grid-template-columns: [label] minmax(100px, 10%) [field] minmax(calc(100% - 100px - 0.3em), calc(90% - 0.3em));
```

Comment:
Where does the `100px` come from? Should we convert it to a variable?

Reply:
Sure, here you have 2 screenshots with a small and wide screen to show it is always aligned and also the magnifier icon is not hidden behind aladin div.
![image](https://user-images.githubusercontent.com/44274704/164310729-29e14cd9-14eb-425e-b522-400e65450c66.png)
![image](https://user-images.githubusercontent.com/44274704/164310834-e78f80f3-9236-4bd7-8235-3a823babb1e0.png)

Reply:
Very nice